frontend/app/components/encryptable/show.js

```diff
@@ -21,6 +21,12 @@ export default class ShowComponent extends Component {
   @tracked
   isFile = this.args.encryptable.isFile;
 
+  @tracked
+  isCredentialSharing = false;
+
+  @tracked isTransferredCredentials =
+    this.args.encryptable.sender_name !== null;
```

Comment:
is this really reliable? doesn't it also apply for transferred file entries?

`hasSender` might also be a possible name

Reply:
In my view, the renaming is not needed, because the variable name says, what value is saved in it?!